py/desispec/resolution.py

```diff
@@ -94,4 +105,32 @@ def to_fits_array(self):
         """
         return self.data
 
-#- (Unit tests moved to desispec.test.test_resolution)
\ No newline at end of file
+def _gauss_pix(x, mean=0.0, sigma=1.0):
```

Comment:
Since this is an implementation support routine and there is no obvious rationale for mean != 0 in a resolution context, I would probably drop the mean arg here.  If you want to keep it, then you should add a unit test for mean != 0.